### PR TITLE
Add Dashboard Sharing info to Site Health

### PR DESCRIPTION
**Pull request by asvinb**
## Summary

Addresses issue:

- #4534 

## PR Author Checklist

- [x] My code is tested and passes existing unit tests.
- [x] My code has an appropriate set of unit tests which all pass.
- [x] My code is backward-compatible with WordPress 4.7 and PHP 5.6.
- [x] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [x] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Code Reviewer Checklist

- [x] Run the code.
- [x] Ensure the acceptance criteria are satisfied.
- [x] Reassess the implementation with the IB.
- [x] Ensure no unrelated changes are included.
- [x] Ensure CI checks pass.
- [ ] Check Storybook where applicable.
- [x] Ensure there is a QA Brief.

## Merge Reviewer Checklist

- [ ] Ensure the PR has the correct target branch.
- [ ] Double-check that the PR is okay to be merged.
- [ ] Ensure the corresponding issue has a ZenHub release assigned.
- [ ] Add a changelog message to the issue.

### CODE REVIEW

File: includes/Core/Util/Debug_Data.php [hunk 1]
```diff
@@ -363,6 +364,120 @@ private function get_active_modules_field() {
...
+			'debug' => join( ', ', $role_slugs ),
+		);
+	}
+
+	/**
+	 * Gets the user friendly and debug values for module management used in module_sharing_settings.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param string $management The module management value. Can be wither `owner` or `all_admins`.
```
Comment (jimmymadon) [inline includes/Core/Util/Debug_Data.php:462]:
```suggestion
	 * @param string $management The module sharing settings management value. Can be either `owner` or `all_admins`.
```

File: includes/Core/Util/Debug_Data.php [hunk 2]
```diff
@@ -363,6 +364,120 @@ private function get_active_modules_field() {
...
+	private function get_module_management( $management ) {
+		switch ( $management ) {
+			case 'all_admins':
+				return array(
+					'value' => __( 'Any admin signed in with Google', 'google-site-kit' ),
+					'debug' => 'all_admins',
+				);
+			default:
+				return array(
+					'value' => __( 'Only me', 'google-site-kit' ),
```
Comment (jimmymadon) [inline includes/Core/Util/Debug_Data.php:475]: As per the AC, this string should be:
```suggestion
					'value' => __( 'Owner', 'google-site-kit' ),
```

**Review by jimmymadon — APPROVED** [review]
This LGTM @asvinb! There are two very minor typos which you can address and move to MR directly. I will approve this to save some back and forth! Thank you!

File: includes/Core/Util/Debug_Data.php (hunk 1 above)
Comment (asvinb) [reply includes/Core/Util/Debug_Data.php]: Thanks for catching that @jimmymadon ! Updated!

File: includes/Core/Util/Debug_Data.php (hunk 2 above)
Comment (jimmymadon) [reply includes/Core/Util/Debug_Data.php]: As discussed on our call, in the case where the Site Health info is for an admin who is _not_ the owner of the shared modules, "Only Me" is technically incorrect as a value for management, hence "Owner" is better.

**Review by tofumatt — APPROVED** [review]

**Pull request merged**